PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsHostConfigurationViewModel.kt

```diff
@@ -186,7 +194,7 @@ class AzureFunctionsHostConfigurationViewModel(
         this.trackProjectArguments = trackProjectArguments
         this.trackProjectWorkingDirectory = trackProjectWorkingDirectory
         runnableProjectsModel.projects.adviseOnce(lifetime) { projectList ->
-            val mappedProjectList = projectList.map(AzureFunctionsRunnableProjectUtil::patchRunnableProjectOutputs).toList()
+            val mappedProjectList = projectList //.map(AzureFunctionsRunnableProjectUtil::patchRunnableProjectOutputs).toList()
```

Comment:
Dead code? If yes - no need to copy projectList into `mappedProjectList` variable

Reply:
Dead code indeed. Updated.